src/media-gallery/media-gallery.js

```diff
@@ -123,16 +135,38 @@ component.template = `
             <span class="media-gallery__screen-reader-only">Remove</span>
         </button>
     </div>
+    <div class="media-list__media-container" ng-if="vm.view === vm.viewPreferences.LIST"
+         ng-repeat="media in vm.media">
+        <div class="col-md-12 media-list__row">
+            <div class="col-md-1" ng-click=""vm.viewLarger(media)">
+                <div class="media-list__image" 
+                     ng-style="{'background-image': 'url(' + media.imageUrl + ')' }"> 
+                    <img class="media-gallery__screen-reader-only" 
+                         ng-src="{{:: media.imageUrl}}" 
+                         alt="{{:: media.title}}">
+                </div>
+            </div>
+            <div class="col-md-10 media-list__file-name" ng-click="vm.viewLarger(media)"></div>
+            <div class="col-md-1"><a href="#" class="media-list__delete svgicon--trash-can btn--icon" ng-click="vm.removeMedia(media)" alt="" title=""></a></div>
+        </div>
+        <div class="col-md-12 editable-panel__divider media-list__divider"></div>
+    </div>
 </div>
 `;
 
 component.controller = function MediaGalleryController() {
     const vm = this;
 
+    vm.viewPreferences = {
+        TILE: 'Tile',
```

Comment:
Perhaps you can move the constant out of the controller and as an angular constant.